Makefile.toml

```
@@ -0,0 +1,68 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+TARGET = { value = "${CARGO_MAKE_RUST_TARGET_TRIPLE}", condition = { env_not_set = [
+    "TARGET",
+] } }
+
+[tasks.format]
+install_crate = { crate_name = "rustfmt-nightly", binary = "rustfmt", test_arg = "--help" }
+command = "cargo"
+args = ["+nightly", "fmt", "--", "--check"]
+
+[tasks.clippy]
+install_crate = "cargo-clippy"
+command = "cargo"
+args = ["clippy"]
+
+[tasks.check]
+command = "cargo"
+args = ["check"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
+
+[tasks.ziglang]
+script = '''
+#!/usr/bin/env bash
+if ! pip3 show ziglang &> /dev/null; then
```

If zig is already installed using something that is not pip, then one should not trigger a pip install.

```suggestion
if ! zig --help &> /dev/null; then
```